### DISCUSSION TITLE
Activity feed prompts me to follow myself. It fails with an error message

### DISCUSSION THREAD

**Discussion by qw-in** [community/community#21046] [Mobile] [answered]
![IMG_20220721_170111.jpg](https://user-images.githubusercontent.com/19194187/180334378-b3f48b56-8d5f-438d-91ef-4585cf618cd7.jpg)

**Comment by iskae** [comment, marked as answer]
Hi @qw-in 👋 

Thanks for reaching out and for your bug report ❤️ 

It appears that we have already rolled out a couple of bugfixes for this behaviour 🎉 It should be fixed in the near future 🙇 

Cheers!
